core/schema/container.graphqls

```diff
@@ -74,7 +74,11 @@ type Container {
     """
     Secrets to pass to the build.
 
-    They will be mounted at /run/secrets/[secret-name].
+    They will be mounted at /run/secrets/[secret-name] in the build container
+
+    They can be accessed in the Dockerfile using the "secret" mount type
+    and mount path /run/secrets/[secret-name]
+    e.g. RUN --mount=type=secret,id=my-secret curl url?token=$(cat /run/secrets/my-secret)"
```

Comment:
`./hack/make sdk:all:generate` to generate sdks with changes

Reply:
Tried this but although the Go, Node SDKs are regenerated, the Elixir one fails with an error

```
194: exec mix local.hex --force ERROR: process "mix local.hex --force" did not complete successfully: exit code: 255
194: > in sdk > elixir > generate
```

As a result, many Elixir files have been deleted (refer PR file list). How should this be fixed?

Reply:
@TomChv note also that even after running the above command, the CI tests are still failing for Go, Python etc with the same error as before. Please let me know any ideas to fix this as well.

Reply:
That's curious, I have never hit this issue before.

And it is normal you didn't pushed python and node version